include/oneapi/dpl/internal/common_config.h

```diff
@@ -19,7 +19,11 @@
 #ifndef _MSVC_LANG
 #    define _ONEDPL___cplusplus __cplusplus
 #else
-#    define _ONEDPL___cplusplus _MSVC_LANG
+#    if _MSVC_LANG > __cplusplus
+#        define _ONEDPL___cplusplus _MSVC_LANG
+#    else
+#        define _ONEDPL___cplusplus __cplusplus
```

Comment:
Please see details in [](https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170#standard-predefined-macros) :